src/DetailsView/tab-stops-requirement-instances-collapsible-content.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+    CheckboxVisibility,
+    ColumnActionsMode,
+    ConstrainMode,
+    DetailsList,
+    IColumn,
+    Icon,
+    Link,
+} from 'office-ui-fabric-react';
+import * as React from 'react';
+import * as styles from './tab-stops-requirement-instances-collapsible-content.scss';
+
+export type TabStopsRequirementInstancesCollapsibleContentProps = {
+    instances: TabStopsRequirementResultInstance[];
+};
+export const TabStopsRequirementInstancesCollapsibleContent =
+    NamedFC<TabStopsRequirementInstancesCollapsibleContentProps>(
+        'TabStopsRequirementInstancesCollapsibleContent',
+        props => {
+            const onRenderCapturedInstanceDetailsColumn = (
+                instance: TabStopsRequirementResultInstance,
+            ): JSX.Element => {
+                return (
+                    <AssessmentInstanceDetailsColumn
+                        background={'#767676'}
+                        textContent={instance.description}
+                        headerText={'Comment:'}
+                        tooltipId={instance.id}
+                    />
+                );
+            };
+
+            const onRenderCapturedInstanceIconsColumn = (
+                instance: TabStopsRequirementResultInstance,
+            ): JSX.Element => {
+                return (
+                    <>
+                        <Link className={styles.editButton} onClick={() => {}}>
+                            <Icon iconName="edit" ariaLabel={'edit instance'} />
+                        </Link>
+                        <Link className={styles.removeButton} onClick={() => {}}>
+                            <Icon iconName="delete" ariaLabel={'delete instance'} />
+                        </Link>
+                    </>
+                );
+            };
+
+            const columns: IColumn[] = [
+                {
+                    key: 'failureDescription',
+                    name: 'Failure description',
+                    fieldName: 'description',
+                    minWidth: 200,
+                    maxWidth: 400,
+                    isResizable: true,
+                    onRender: onRenderCapturedInstanceDetailsColumn,
+                    columnActionsMode: ColumnActionsMode.disabled,
+                },
+                {
+                    key: 'instanceActionButtons',
+                    name: 'instance actions',
+                    isIconOnly: true,
+                    fieldName: 'instanceActionButtons',
+                    minWidth: 100,
+                    maxWidth: 100,
+                    isResizable: false,
+                    onRender: onRenderCapturedInstanceIconsColumn,
+                    columnActionsMode: ColumnActionsMode.disabled,
+                },
+            ];
+
+            return (
+                <DetailsList
+                    items={props.instances}
+                    columns={columns}
+                    checkboxVisibility={CheckboxVisibility.hidden}
```

Comment:
This is interesting. I need to make this change in the requirements table! I just used a string but this seems more ideal.